src/app/models/propTypes/inline/index.js

```diff
@@ -1,11 +1,15 @@
-import { string, arrayOf, oneOf, shape } from 'prop-types';
+import { string, arrayOf, oneOfType, shape } from 'prop-types';
```

Comment:
I changed this after reading this [article on oneOf vs oneOfType](https://jaketrent.com/post/react-oneof-vs-oneoftype/)